CHANGES.rst

```diff
@@ -2,10 +2,11 @@
 CHANGES
 =======
 
-5.0.1 (unreleased)
-------------------
+5.1 (unreleased)
```

Comment:
I am constantly working to get rid of these feature version numbers ending in `.0`

In my opinion the version number of the unreleased version should always be the next feature version as we are developing towards the next feature not towards the next bug fix
Using `zest.releaser` for the release always requires manual changes when using a version number consisting of three parts.